src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +49,41 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        return (dateParams.length == DATE_PARAMS && dateParams[1] <= MAX_MONTH)
```

Comment:
let's remove this check  and just catch exception and return Optional.empty() after catch block

Reply:
done